feat(changelog): audit-trail-added-account-management-actions-in-audit-t 2025-07-09 (#5249)
scaleway/docs-content: changelog/july2025/2025-07-09-audit-trail-added-account-management-actions-in-audit-t.mdx

---
title: Account management actions in Audit Trail
status: added
date: 2025-07-09
category: monitoring
product: audit-trail
---

Account management activity can now be tracked on Audit Trail.

> Read [our documentation](https://www.scaleway.com/en/docs/audit-trail/reference-content/adt-supported-endpoints/) to see the list of actions that appear. Account actions are tracked with the Paris region.
> 

Reviewer: ```suggestion
Read [our documentation](/audit-trail/reference-content/adt-supported-endpoints/) to see the list of actions available. Account actions are tracked in the Paris region.
```
